src/lib/trajectory/plotting.ts

```
@@ -0,0 +1,200 @@
+  for (const key of all_keys) {
+    // Skip Step for x-axis (we use step index instead)
+    if (key === `Step`) continue
+    // Skip the marker property used for lattice parameter detection
+    if (key === `_constant_lattice_params`) continue
+
+    const x_values: number[] = []
+    const y_values: number[] = []
+
+    for (let idx = 0; idx < all_extracted_data.length; idx++) {
+      const data = all_extracted_data[idx]
+      if (key in data && typeof data[key] === `number`) {
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)